content/posts/docker-prune.md

```diff
@@ -30,6 +41,8 @@ The "Active" column represents objects that are associated with running containe
 
 Docker is eating up roughly 23.5 GB of storage capacity. 23 GB of that is reclaimable (or unused). Yikes! Let's free that up.
 
+[//]: # (what is a typical size? I heard about 5GB. Any idea based on your experience?)
```

Comment:
Typical size of what?

I'm not talking about a single Docker image. All Docker objects combined on my laptop is taking up 23.5 GB or storage space.